### DISCUSSION TITLE
Option for Copying Github Repo in Clipboard

### DISCUSSION THREAD

**Discussion by noob6t5** [community/community#146048] [Mobile]
Any Option for Copying Github Repo into Clipboard would be great.

**Comment by tyagraj777** [comment]
great question, thanks for posting 

Here's a small implementation using HTML, JavaScript, and a mockup design for a GitHub-like interface. This example demonstrates how you might add a "Copy URL" button.

**HTML + CSS + JavaScript Implementation**

PFA sample implementation for reference test23.html

please ref the same and get the task done

Cheers !!!
[test23.html.txt](https://github.com/user-attachments/files/17988127/test23.html.txt)

**Reply by noob6t5** [reply]
How can i use that in GitHub App?

**Reply by tyagraj777** [reply]
How to Use:

1. Save this code as index.html.
2. Open it in your browser to see the functionality in action.
3. Replace the URL inside the value attribute of the <input> element with your repository URL.

**Reply by tyagraj777** [reply]
**HTML Structure:**

A container displays the repository name and URL.
An input field (readonly) shows the repository URL.
A button triggers the copy action.

**CSS Styling:**

The layout is styled for a clean and modern look.
Button hover effects improve the user experience.

**JavaScript Logic:**

When the button is clicked, the repository URL is selected, and the browser's navigator.clipboard API is used to copy it to the clipboard.
A success message appears briefly to confirm the action.

**Mobile Compatibility:**

setSelectionRange(0, 99999) ensures the URL is selected correctly on mobile devices.

**Reply by noob6t5** [reply]
The main thing is i don't want to leave GitHub App .
Like Copy Features within app
